samples/consul-ui/js/nodes.js

```diff
@@ -94,11 +94,11 @@ class ConsulNodes {
       content.setAttribute('class','instance-content');
       var contentHead = document.createElement('div');
       contentHead.setAttribute('class','instance-content-header');
-      contentHead.appendChild(nodeNameGenator(instance['Node']['Name']));
+      contentHead.appendChild(nodeNameGenator(instance['Node']['Name'],instance['Node']['Address']));
       contentHead.appendChild(nodeAddressGenator(instance['Node']['Address']));
       contentHead.appendChild(nodeMetaGenator(instance['Node']['Meta']));
       content.appendChild(contentHead);
-      content.appendChild(servicesGenerator(instance['Service'], instance['Node']['Name']));
+      content.appendChild(servicesGenerator(instance['Service'], instance['Node']['Name'], instance['Node']['Address']));
```

Comment:
For servicesGenerator, use `instance.service_address`, not `instance['Node']['Address'])`

=> it will use instance['Service']['Address'] if specified, instance['Node']['Address'] otherwise

Reply:
Done